cyences_app_for_splunk/default/data/ui/views/cs_asset_intelligence.xml

```diff
@@ -195,6 +216,27 @@
       </table>
     </panel>
   </row>
+  <row depends="$tkn_show_hide_tenable$">
+    <panel>
+      <title>Network Traffic from Vulnerable Ports (Detected from Tenable)</title>
+      <input type="time" token="tkn_timeRange_tenable">
+        <label></label>
+        <default>
+          <earliest>-24h@h</earliest>
+          <latest>now</latest>
+        </default>
+      </input>
+      <viz type="missile_map.missile_map">
+        <search>
+          <query>| tstats count from datamodel=Network_Traffic where [| inputlookup cs_tenable_vuln | search ($tkn_filter_ip_host$ AND (vul_state="open" OR vul_state="reopened")) | stats count by ip vul_port | search vul_port!=0 | eval field = "(All_Traffic.src_ip = \"".ip."\" AND All_Traffic.src_port = \"".vul_port."\") OR (All_Traffic.dest_ip = \"".ip."\" AND All_Traffic.dest_port = \"".vul_port."\")" | appendpipe [| stats count | where count==0 | eval field="false()" | fields field] | fields field | return 10000 $field ] by All_Traffic.dest_ip All_Traffic.dest_port All_Traffic.src_ip All_Traffic.src_port | rename All_Traffic.* as * | eventstats min(count) as min_value, max(count) as max_value | eval weight=(5*(count-min_value))/(max_value-min_value)+1 , animate="true" | appendpipe [| inputlookup cs_tenable_vuln | search ($tkn_filter_ip_host$) | fields ip | stats values(ip) as ip] | eventstats values(ip) as ip | eval pulse_at_start = if(isnotnull(mvfind(src_ip, ip)),"true","false") | eval pulse_at_end = if(isnotnull(mvfind(dest_ip, ip)),"true","false")| eval color= "#dc4e41" | iplocation src_ip | rename lat as start_lat, lon as start_lon | iplocation dest_ip | rename lat as end_lat, lon as end_lon | `cs_network_filter_internal_traffic`</query>
```

Comment:
| search vul_port!=0
bring above part of the query before stats for performance.

Reply:
Follow all the other comments from Qualys query.